src/components/HomePage/SearchArea/AdvancedSearch/TechOptions.js

```diff
@@ -1,9 +1,10 @@
 /* istanbul ignore file */
 
-export default Object.freeze([
-    { label: "React.js", value: "REACT_JS" },
-    { label: "Node.js", value: "NODE_JS" },
-    { label: "Python", value: "PYTHON" },
-    { label: "Java", value: "JAVA" },
-    { label: "C++", value: "C++" },
-]);
+export default Object.freeze({
+    "REACT_JS": "React.js",
```

Comment:
These keys are also different to the ones on "TechnologyTypes.js" from the backend. I've noticed that in the backend the Technology types are not all capital letters, while the Field types are all Capital letters. I am not sure if it is intended.

Reply:
This is not in the best state right now, but this issue will attempt at making it all better NIAEFEUP/nijobs-be#103 by making both projects sync on these values